packages/grid/_modules_/grid/components/toolbar/DensitySelector.tsx

```diff
@@ -25,17 +25,17 @@ export function DensitySelector() {
   const DensityOptions: Array<DensityOption> = [
     {
       icon: <DensityCompactIcon />,
-      label: 'Compact',
+      label: apiRef!.current.getText('densityOptionCompact') as string,
```

Comment:
Could we remove as string?

Reply:
This is just a type fix as the Label receives either a string or undefined.

Reply:
Mind that sometimes, we might want to allow React nodes, not just string

Reply:
Done